db/redis.js

```diff
@@ -64,7 +64,7 @@ async function addDoorTestSensorData(locationid, signal) {
 async function addIM21DoorSensorData(locationid, doorSignal) {
   // eslint-disable-next-line eqeqeq
   if (doorSignal == SESSIONSTATE_DOOR.CLOSED || doorSignal == SESSIONSTATE_DOOR.OPEN) {
-    await client.xadd(`door:${locationid}`, '*', 'signal', doorSignal)
+    await client.xadd(`door:${locationid}`, 'MAXLEN', '~', '10000', '*', 'signal', doorSignal)
   }
 }
 
```

Comment:
I see that you didn't add a MAXLEN to `addVitals`, `addDoorSensorData`, or `addDoorTestSensorData`. I could see leaving the door ones out because they are being removed in the INS PR soon, but why not do `addVitals`?

Reply:
oversight!